Projects/UOContent/Commands/Possess.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using Server.Commands.Generic;
+using Server.Mobiles;
+
+namespace Server.Commands
+{
+    internal static class Swap
+    {
+        private static readonly List<Layer> SPECIAL_LAYERS = new() { Layer.Backpack, Layer.Bank, Layer.Mount };
+
+        private static List<Item> GetItemsFromLayers(IEnumerable<Item> items) =>
+            new List<Item>(items).FindAll(item => !SPECIAL_LAYERS.Contains(item.Layer));
+        public static void Items(Mobile target, PlayerMobile caster)
+        {
+            var targetItems = GetItemsFromLayers(target.Items);
+            var casterItems = GetItemsFromLayers(caster.Items);
+
+            foreach (Item item in casterItems)
+            {
+                caster.RemoveItem(item);
+            }
+
+            foreach (Item item in targetItems)
+            {
+                caster.EquipItem(item);
+                target.RemoveItem(item);
+            }
+
+            foreach (Item item in casterItems)
+            {
+                target.EquipItem(item);
+            }
+        }
+
+        public static void Titles(Mobile target, PlayerMobile caster)
+        {
+            var casterTitle = caster.Title;
+            caster.Title = target.Title;
+            target.Title = casterTitle;
+        }
+    }
+
+    public class PossessCommand : BaseCommand
+    {
+        public PossessCommand()
+        {
+            AccessLevel = AccessLevel.GameMaster;
+            Supports = CommandSupport.AllMobiles;
+            Commands = new[] { "Possess" };
+            ObjectTypes = ObjectTypes.Mobiles;
+            Usage = "Possess";
+            Description = "Takes control of a mobile";
+        }
+
+        public override void Execute(CommandEventArgs e, object obj)
+        {
+            Console.WriteLine("Executing possess command");
+
+            if (e.Mobile is PlayerMobile caster && obj is Mobile target)
```

Comment:
We might want to exclude some mobile types?
* pets
* summons
* hireables?
* vendors?
* questors?
* guards?